log/httpLogger.go

```diff
@@ -0,0 +1,39 @@
+package log
+
+import (
+	"github.com/golang/glog"
+	"github.com/hashicorp/go-retryablehttp"
+)
+
+var _ retryablehttp.LeveledLogger = retryableHTTPLogger{}
+
+type retryableHTTPLogger struct {
+}
+
+func NewRetryableHTTPLogger() retryablehttp.LeveledLogger {
+	return retryableHTTPLogger{}
+}
+
+func (r retryableHTTPLogger) Error(msg string, keysAndValues ...interface{}) {
+	if glog.V(3) {
+		LogNoRequestID(msg, keysAndValues...)
+	}
+}
+
+func (r retryableHTTPLogger) Warn(msg string, keysAndValues ...interface{}) {
+	if glog.V(4) {
+		LogNoRequestID(msg, keysAndValues...)
+	}
+}
+
+func (r retryableHTTPLogger) Info(msg string, keysAndValues ...interface{}) {
+	if glog.V(5) {
+		LogNoRequestID(msg, keysAndValues...)
+	}
+}
+
+func (r retryableHTTPLogger) Debug(msg string, keysAndValues ...interface{}) {
+	if glog.V(6) {
+		LogNoRequestID(msg, keysAndValues...)
```

Comment:
`LogNoRequestID()` uses `kitlog` which supports [level-filtering](https://pkg.go.dev/github.com/go-kit/log/level) but unsure why we use this lib. `glog` seems much more widely used.

Reply:
yeah.. it does look odd, i'll see if there's a better way but glog didn't seem to have the same function signature with `msg string, keysAndValues ...interface{}`